docs/codelabs/building-beautiful-sites/docsite/1-setup.md

```diff
@@ -17,28 +17,10 @@ edited for a non-technical audience
 Duration: 5:00
 {: .duration}
 
-## Download the starter codelab app
-
-[Download starter app](https://github.com/material-components/material-components-web/archive/master.zip)
-
-> The starter app is located within the `docs/codelabs/building-beautiful-sites/starter directory`. Be sure to `cd` into that directory before beginning.
-{: .hint}
-
-## Or clone it from GitHub
-
-```shell
-git clone https://github.com/material-components/material-components-web
-cd material-components-web/docs/codelabs/building-beautiful-sites/starter
-```
-
-> More help: [Cloning a repository from GitHub](https://help.github.com/articles/cloning-a-repository/)
-{: .hint}
 
 ## Install the project’s dependencies
 
-We will be using [live-server](https://www.npmjs.com/package/live-server) module as our local web development server so we can serve our site as if it were running in a production web application. Live-server automatically updates the web page whenever any HTML, CSS, or JavaScript changes are made. There is no refreshing needed! Live-server is listed as a [devDependency](https://docs.npmjs.com/files/package.json#devdependencies) for this project, which means we can install it using npm.
-
-From the starter directory (it should be your current directory if you follow the above step), run `npm install`. You will see a lot of activity and at the end, the output will show a successful install.
+From the starter directory (it should be your current directory), run `npm install`. You will see a lot of activity and at the end, the output will show a successful install.
 
 ## Run the starter app
 
```

Comment:
Line 27: This one is totally my mistake from the last one, but could you add backticks for `npm install`

Reply:
Line 32: also not part of your change, but nit: replace Voila with Voilà

Reply:
Done